docs/cql/ddl.rst

```diff
@@ -109,9 +109,12 @@ For example:
 
 .. TODO Add a link to the description of minimum_keyspace_rf when the ScyllaDB options section is added to the docs.
 
-You can configure the minimum acceptable replication factor using the ``minimum_keyspace_rf`` option. 
-Attempting to create a keyspace with a replication factor lower than the value set with 
-``minimum_keyspace_rf`` will return an error (the default value is 0). 
+* The replication factor configured for a DC should be equal to or higher than
+  the number of nodes in that DC. Configuring a lower RF may prevent creating
+  tables in that keyspace. 
+* You can configure the minimum acceptable replication factor using the ``minimum_keyspace_rf`` option. 
+  Attempting to create a keyspace with a replication factor lower than the value set with 
+  ``minimum_keyspace_rf`` will return an error (the default value is 0). 
```

Comment:
Sadly, I don't think we ever supported this option.
@tgrabiec please stand me corrected...

Reply:
`minimum_keyspace_rf` seems to be referenced only in docs. There is `minimum_replication_factor_warn_threshold`, which is set to 3, and we warn, not fail.